docs/getting-started/csm-stack/csm-sandbox.ipynb

```diff
@@ -0,0 +1,257 @@
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "99f757a292504ab19d2b8db31cbd9cd3",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(FloatSlider(value=0.0, description='Focal Length', layout=Layout(width='600px'), max=0.2…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# clear the plot from any previous footprints/plots\n",
+    "plt.close()\n",
+    "\n",
+    "# Copy the ISD Dictionary, we will modify it and compare to the original.\n",
+    "isd_dict_mod = copy.deepcopy(isd_dict)\n",
+    "\n",
+    "print('Adjust Sliders to add or subtract from the values at the following ISD Keys:')\n",
+    "\n",
+    "# Slider Widgets\n",
+    "wide_lay  = Layout(width='600px')\n",
+    "wide_desc = {'description_width': '150px'}\n",
+    "@widgets.interact(\n",
+    "        fl_add=widgets.FloatSlider(min=-0.25, max=.25, step=0.001, description='Focal Length', layout=wide_lay, style=wide_desc), \n",
+    "        dcl_add=widgets.FloatSlider(min=-4, max=4, step=0.05, description='Detector Center Line', layout=wide_lay, style=wide_desc),\n",
+    "        dcs_add=widgets.FloatSlider(min=-2.5, max=2.5, step=0.02, description='Detector Center Sample', layout=wide_lay, style=wide_desc),\n",
+    "        opt_x=widgets.FloatSlider(min=-0.001, max=0.001, step=0.00001, description='Optical Distortion X', layout=wide_lay, style=wide_desc, readout_format='.5f'),\n",
+    "        opt_y=widgets.FloatSlider(min=-2e-6, max=2e-6, step=2e-8, description='Optical Distortion Y', layout=wide_lay, style=wide_desc, readout_format='.1e'),\n",
+    "        opt_z=widgets.FloatSlider(min=-1e-8, max=1e-8, step=1e-10, description='Optical Distortion Z', layout=wide_lay, style=wide_desc, readout_format='.1e'),\n",
+    "        ect_add=widgets.FloatSlider(min=-1e-2, max=1e-2, step=1e-4, description='Exposure (Center) Time', layout=wide_lay, style=wide_desc, readout_format='.4f')\n",
+    "    )\n",
+    "# This function executed whenever one of the slider widgets is adjusted\n",
+    "def exec_widget_function(fl_add, dcl_add, dcs_add, opt_x, opt_y, opt_z, ect_add):\n",
+    "\n",
+    "    # If you're curious where the ISD values came from, \n",
+    "    # Detector Center was from NAIF Boresight Line/Sample\n",
+    "    # Optical Distortion was from NAIF OD_K\n",
+    "    # ISIS uses the NAIF Keywords, but Knoten CSM uses other derived ISD values.\n",
+    "\n",
+    "    new_values = {\n",
+    "        'focal_length_model': {'focal_length': 352.9271664 + fl_add},\n",
+    "        'detector_center':    {'line': 0.430442527 + dcl_add, 'sample': 2542.96099 + dcs_add},\n",
+    "        'optical_distortion': {'radial': {'coefficients': [-0.007343 + opt_x, 2.838e-05 + opt_y, 1.284e-08 + opt_z]}},\n",
+    "        'center_ephemeris_time': 297088762.2425226 + ect_add\n",
```

Comment:
this slider is good, but this is changing there center time, not exposure time. Exposure time is part of the `line_scan_rate`. 

Additionally, might be worth adding sliders or changing values in `focal2pixel_lines` and `focal2pixel_samples`